src/components/VtkObliqueView.vue

```diff
@@ -0,0 +1,681 @@
+<template>
+  <div
+    class="vtk-container-wrapper"
+    tabindex="0"
+    @pointerenter="hover = true"
+    @pointerleave="hover = false"
+    @focusin="hover = true"
+    @focusout="hover = false"
+  >
+    <div class="vtk-gutter">
+      <v-btn dark icon size="medium" variant="text" @click="enableResizeToFit">
+        <v-icon size="medium" class="py-1">mdi-camera-flip-outline</v-icon>
+        <v-tooltip
+          location="right"
+          activator="parent"
+          transition="slide-x-transition"
+        >
+          Reset Camera
+        </v-tooltip>
+      </v-btn>
+      <slice-slider
+        class="slice-slider"
+        :slice="currentSlice"
+        :min="sliceMin"
+        :max="sliceMax"
+        :step="1"
+        :handle-height="20"
+        @input="setSlice"
+      />
+    </div>
+    <div class="vtk-container" :class="active ? 'active' : ''">
+      <div class="vtk-sub-container">
+        <div class="vtk-view" ref="vtkContainerRef" />
+      </div>
+      <div class="overlay-no-events tool-layer" ref="toolContainer">
+        <pan-tool :view-id="viewID" />
+        <zoom-tool :view-id="viewID" />
+        <reslice-cursor-tool :view-id="viewID" :view-direction="viewDirection" />
+        <slice-scroll-tool :view-id="viewID" />
+        <window-level-tool :view-id="viewID" />
+      </div>
+      <view-overlay-grid class="overlay-no-events view-annotations">
+        <template v-slot:bottom-left>
+          <div class="annotation-cell">
+            <div>Slice: {{ currentSlice }}/{{ sliceMax }}</div>
+            <div
+              v-if="
+                typeof windowWidth === 'number' &&
```

Comment:
`windowWidth` seems to be defined as a `ComputedRef<Nullable<number>>`. Why do you need to check its type here? Can't you check it is not null?